docs/outlook/online-meeting.md

```diff
@@ -0,0 +1,182 @@
+---
+title: How to create an Outlook mobile add-in for an online-meeting provider (preview)
+description: Discusses how to set up an Outlook mobile add-in for an online-meeting service provider.
+ms.topic: article
+ms.date: 04/09/2020
+localization_priority: Normal
+---
+
+# Create an Outlook mobile add-in for an online-meeting provider (preview)
+
+Setting up an online meeting is a core experience for an Outlook user, and it's easy to [create a Teams meeting with Outlook](/microsoftteams/teams-add-in-for-outlook) mobile. However, creating an online meeting in Outlook with a non-Microsoft service can be cumbersome. By implementing this feature, service providers can make the online meeting creation experience seamless for their Outlook add-in users.
+
+> [!NOTE]
+> This feature is only supported in [preview](../reference/objectmodel/preview-requirement-set/outlook-requirement-set-preview.md) on Android with an Office 365 subscription.
+
+## Scenario: Online-meeting provider
```

Comment:
SEO and clarity: Consider removing H2 ## "Scenario: online meeting provider" and moving everything up a heading level so there's no heading 4